packages/renderer/src/lib/ui/PasswordInput.svelte

```diff
@@ -30,7 +30,7 @@ async function onShowHide() {
   class={$$props.class || ''}
   id="password-{id}"
   name="password-{id}"
-  placeholder="Password"
+  placeholder="password"
```

Comment:
looks like unrelated

Reply:
Password is a shared component that had a capitalized Password placeholder, but it is only used on this page so far and the design has lowercase placeholders for username and password so I changed it here. I don't have a strong opinion (other than the cases should match) but lower looked slightly better to me.